woocommerce/woocommerce-ios
# [Order form] Add shimmering effect to Order and Product values when Order is syncing

## Description

Add shimmering effect to Order and Product values when Order is syncing.

While working on https://github.com/woocommerce/woocommerce-ios/pull/13305 PR I noticed the delay between pricing updates when changing Order attributes or increasing the Quantity. Product or Orer prices can appear momentarily wrong while ORder is syncing.

Previously closed PR [[Order form] Show order item data (not product data) where possible in order form](https://github.com/woocommerce/woocommerce-ios/pull/11431) started the work on show shimmering effect for Product and Order prices that I'm finishing in this PR.

## Testing information


1. Go to the Orders tab
2. Tap on + to create an order
3. Add different types of products
4. Expand Order Payment section
5. Increase any product quantity
6. Confirm that product and order payment section values are shimmering while order is syncing

## Screenshots

https://github.com/user-attachments/assets/61ceaf0b-86fb-48ad-82a8-4f523f9a51b5

---
